app/views/users/join.ejs

```diff
@@ -15,7 +15,9 @@ this license in a file with the distribution.
 	</header>
 	<!-- current text on the join page for liquid  -->
 	<p>
-		#TODO: @tommypacker fill this in/give us the text to fill this in.
+		Joining ACM is completely free! Once you've found your niche in ACM or one of its SIGs, becoming an official member is easy. To become an official member, you need to fill out the quick and easy form below.
```

Comment:
Can you format this with new lines in the source, will be easier to read

Reply:
Also for content, what happens when they fill out the form, are then added to mailing lists?